test/cppsuite/tests/run.cxx

```diff
@@ -87,7 +88,8 @@ print_help()
     std::cout << "\t-h Output a usage message and exit." << std::endl;
     std::cout << "\t-C Configuration. Cannot be used with -f." << std::endl;
     std::cout << "\t-f File that contains the configuration. Cannot be used with -C." << std::endl;
-    std::cout << "\t-l Trace level from 0 (default) to 2." << std::endl;
+    std::cout << "\t-l Trace level from 0 to 3. 1 is the default level all warnings";
+    std::cout << " and errors are logged." << std::endl;
```

Comment:
This can be added in line 91 ?

Reply:
It ran over so I put it on a newline, but are you suggesting I just remove the second `std::cout <<` pair.

Reply:
I was thinking:
std::cout << "\t-l Trace level from 0 to 3. 1 is the default level all warnings and errors are logged.: << std::endl;

Reply:
Too long for one line.

Reply:
I think you meant this: 
```
    std::cout << "\t-l Trace level from 0 to 3. "
        "1 is the default level all warnings and errors are logged." << std::endl;
```
Which is sufficient

Reply:
yes after s_all you should get something like this.

Reply:
5abf1d1c6